internal/sqladapter/testing/adapter.go.tpl

```diff
@@ -117,7 +117,7 @@ func TestInsertReturning(t *testing.T) {
 	assert.NotZero(t, itemMap["id"], "Must not be zero after inserting")
 
 	itemStruct := struct {
-		ID int `db:"id,omitempty"`
```

Comment:
hmm.. possibly calling it _adapter.tpl.go could work.. then on generation you make adapter.go